src/java/org/apache/cassandra/db/virtual/AbstractVirtualTable.java

```diff
@@ -42,7 +42,7 @@
  */
 public abstract class AbstractVirtualTable implements VirtualTable
 {
-    private final TableMetadata metadata;
+    protected final TableMetadata metadata;
```

Comment:
Not that I mind the change, but it isn't necessary for this patch, and there is `metadata()` if it were.